test/common/memory_usage.h

```diff
@@ -119,6 +119,7 @@ namespace utils {
         ASSERT(status == KERN_SUCCESS, NULL);
         return info.virtual_size - shared_size;
 #else
+        utils::suppress_unused_warning(stat);
```

Comment:
Otherwise:
```
In file included from /RAMDisk/oneTBB-2021.1.1/test/tbbmalloc/test_malloc_compliance.cpp:29:
/RAMDisk/oneTBB-2021.1.1/test/common/memory_usage.h: In function 'size_t utils::GetMemoryUsage(utils::MemoryStatType)':
/RAMDisk/oneTBB-2021.1.1/test/common/memory_usage.h:84:42: error: unused parameter 'stat' [-Werror=unused-parameter]
     size_t GetMemoryUsage(MemoryStatType stat = currentUsage) {
                           ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.

```